src/app/static/src/scss/partials/strategise.scss

```diff
@@ -1,33 +1,52 @@
 .strategise {
-  .summary {
-    tr td:first-child {
-      font-weight: 700;
-      white-space: nowrap;
+  .form {
+    .icon-small svg {
+      height: 20px;
+      width: 20px;
+      vertical-align: sub;
+    }
+  }
+
+  .results {
+    .lds-spin {
+      display: block;
     }
+
     .table-success {
       background-color: #bbf0fb;
     }
+
     .table-secondary {
       background-color: #e0fae0;
     }
+
     .table-primary {
       background-color: #dbb8db;
     }
+
     .table-warning {
       background-color: #f5c6cb;
     }
+
     .table-danger {
       background-color: #ffe6b8;
     }
-    .lds-spin {
-      display: block;
-    }
-  }
-  .form {
-    .icon-small svg {
-      height: 20px;
-      width: 20px;
-      vertical-align: sub;
+
+    .summaryTable {
+      tr td:first-child {
+        font-weight: 700;
+        white-space: nowrap;
+      }
+    }
+
+    .detailsTable {
+      tr td:nth-child(2) {
+        white-space: nowrap;
+      }
+
+      tr:last-child {
+        font-weight: bold;
+      }
```

Comment:
Ah nice, I was wondering how you'd done that!

Reply:
The right way to do this is actually by using custom footer row... but the documentation is very patchy and I ran out of patience. The only thing we lose is ability for it to be sticky. Can always revisit.